### PR TITLE
Bugfix for Landing Page's responsive styling

### PR DESCRIPTION
**Pull request by yujuncho**
The following bugs were addressed on the landing page

- Header title's unresponsive, large font-size created undesired white space for mobile devices
- About Us section didn't wrap the photo and text into a single-column layout which also resulted in undesired white space for mobile devices
- Contact section used unnecessary rows which also resulted in undesired white space for mobile devices and also resulted in the social icons to be left-aligned rather than centered
- Underline for About Us and Contact section titles was not centered

### CODE REVIEW

File: client/public/css/style.css [hunk 1]
```diff
@@ -217,6 +216,11 @@ header .intro-text {
   padding-bottom: 100px;
   text-align: center;
 }
+@media (min-width: 768px) {
```
Comment (adbahrani) [inline client/public/css/style.css]: using media is discouraged if you're using bootstrap

File: client/public/css/style.css [hunk 2]
```diff
@@ -465,6 +471,8 @@ header .intro-text {
 }
 #contact form {
   padding-top: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
```
Comment (adbahrani) [inline client/public/css/style.css]: Try to use https://getbootstrap.com/docs/4.0/utilities/spacing/ as much as possible. Refer to this only if you need to do a globe change

File: client/src/components/contact.jsx [hunk 3]
```diff
@@ -8,64 +8,62 @@ export class Contact extends Component {
...
-                            placeholder="Email"
-                            required="required"
-                          />
-                          <p className="help-block text-danger"></p>
-                        </div>
+                <form
+                  name="sentMessage"
+                  id="contactForm"
+                  noValidate
+                  style={{ width: "100%" }}
```
Comment (adbahrani) [inline client/src/components/contact.jsx]: you can use class w-100 too form bootstrap

Comment (yujuncho) [reply client/src/components/contact.jsx]: Updated!

File: client/public/css/style.css (hunk 2 above)
Comment (yujuncho) [reply client/public/css/style.css]: Updated!

File: client/public/css/style.css (hunk 1 above)
Comment (yujuncho) [reply client/public/css/style.css]: Updated to use Bootstrap RFS for responsive text

**Pull request merged**